fridge/utilities/material_smear.py

```diff
@@ -0,0 +1,117 @@
+from FRIDGe.fridge.utilities import material_reader as mr
+import numpy as np
+import pandas as pd
+
+AVOGADROS_NUMBER = 0.6022140857
+
+
+def material_smear(material_wt_per, material_str):
+    """
+        Calculates the atom percent of a material given multiuple materials.
+
+        args:
+            material_wt_per (double array): array with all isotopes present in
+            the material in weight percent
+            material_str (str array): array with the name of each material
+
+        returns:
+            atom_percent (double array): array with all isotopes present in
+                the material in smeared at%
+            atom_density (double): the total atom density of the smeared material
+    """
+    assert material_str
+    for iter, name in enumerate(material_str):
+        if name == 'Void' or name == 'void':
+            material_str.remove('Void')
+            material_wt_per.pop(iter)
+
+    wt_per = material_creator(material_wt_per, material_str)
+    atom_percent, atom_density = smear_wt2at_per(wt_per)
+    return atom_percent, atom_density
+
+
+def material_creator(material_wt_per, material_str):
+    """
+        Calculates the weight percent of a material given multiuple materials.
+
+        args:
+            material_wt_per (double array): array with all isotopes present in
+            the material in weight percent
+            material_str (str array): array with the name of each material
+
+        returns:
+            material_array (double array): array with all isotopes present in
+                the material in smeared wt %
+    """
+    material_array = pd.DataFrame
+    density_array = np.zeros(len(material_str))
+    for iter, material_name in enumerate(material_str):
+        wt_per, material_density = mr.get_final_wt_per([material_name])
+        temp, temp1 = mr.material_reader([material_name])
```

Comment:
I love that material_reader got imported as mr. All these Mr. material_readers sound so formal.